model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -21,6 +21,7 @@ package org.ossreviewtoolkit.model.config
 
 import com.fasterxml.jackson.annotation.JsonAlias
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import org.ossreviewtoolkit.utils.ORT_REPO_CONFIG_FILENAME
```

Comment:
*detekt* should actually complain about the import grouping here.

Reply:
It's fixed.